### PR TITLE
Create train_test_split

### PR DESCRIPTION
**Pull request by harshnandwana**
This  code helps you converting output of PPOCRLabel to feed-able format that you can use for training and separate you single folder and single Label.txt file in train and val split

### CODE REVIEW

**Comment by Evezerest** [comment]
Could you please write instructions in the readme of PPOCRLabel?

**Comment by harshnandwana** [comment]
sure

**Comment by rpsipl2021** [comment]
not working this train_test split
